Examples/Google/SpeechToText/Sources/SpeechService.swift

```diff
@@ -0,0 +1,122 @@
+/*
+ * Copyright 2020, gRPC Authors All rights reserved.
+ *
+ * Licensed under the Apache License, Version 2.0 (the "License");
+ * you may not use this file except in compliance with the License.
+ * You may obtain a copy of the License at
+ *
+ *     http://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing, software
+ * distributed under the License is distributed on an "AS IS" BASIS,
+ * WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+ * See the License for the specific language governing permissions and
+ * limitations under the License.
+ */
+
+import GRPC
+
+typealias Request = Google_Cloud_Speech_V1_StreamingRecognizeRequest
+typealias Response = Google_Cloud_Speech_V1_StreamingRecognizeResponse
+typealias StreamingRecognizeCall = BidirectionalStreamingCall
+
+// Track whether we are currently streaming or not
+enum State {
+  case idle
+  case streaming(StreamingRecognizeCall<Request, Response>)
+}
+
+final class SpeechService {
+  
+  // Generated SpeechClient for making calls
+  private var client: Google_Cloud_Speech_V1_SpeechClient
+  
+  // Track if we are streaming or not
+  private var state: State = .idle
+  
+  init() {
+    precondition(!Constants.apiKey.isEmpty, "Please refer to the README on how to configure your API Key properly.")
+    
+    // Make EventLoopGroup for the specific platform (NIOTSEventLoopGroup for iOS)
```

Comment:
Can you link to https://github.com/grpc/grpc-swift/blob/master/docs/apple-platforms.md as well? I think this file is a nice example of how to _actually_ use gRPC Swift. It's probably worth referring to this file specifically in the README as where the gRPC logic is done.